WordPress/gutenberg
# Hide the deprecation logs under a console group

closes #33148 

While we can't remove these logs for now but core block patterns are supposed to support old WordPress versions as well, we can display the logs in a better way: A collapsed group you can toggle if you want to check the details.

**Testing instructions**

 - Open the editor
 - Open the console
 - Check the collapsed logs.
